src/pages/IstioRuleDetails/IstioRuleInfo.scss

```diff
@@ -0,0 +1,5 @@
+.IstioAceEditor {
```

Comment:
Here again we have a problem of naming homogeneity: in general CSS classes are low-case with dashes. But some of us started with upper camel case, other stick with the norm... Definitely a problem to solve by @aljesusg in KIALI-599 ;-)
[So, no change request here until we have guidelines.]